### ISSUE TITLE
Unhandled promise rejection after importing

### ISSUE THREAD

**Issue by Sorgrum** [zackargyle/react-bash#29]
I used yarn to install the package: `yarn add react-bash`
I'm trying to include it on a page. As soon as I add the import statement `import Terminal from `"react-bash";` I get the following in the console:

```js
Unhandled promise rejection 
TypeError: _react.PropTypes is undefined
Stack trace:
./node_modules/react-bash/lib/component.js@http://0.0.0.0:8000/commons.js:40810:6
__webpack_require__@http://0.0.0.0:8000/commons.js:556:12
fn@http://0.0.0.0:8000/commons.js:87:20
./node_modules/react-bash/lib/index.js@http://0.0.0.0:8000/commons.js:40889:19
__webpack_require__@http://0.0.0.0:8000/commons.js:556:12
fn@http://0.0.0.0:8000/commons.js:87:20
./src/pages/index.js@http://0.0.0.0:8000/commons.js:2876:19
__webpack_require__@http://0.0.0.0:8000/commons.js:556:12
fn@http://0.0.0.0:8000/commons.js:87:20
./.cache/sync-requires.js@http://0.0.0.0:8000/commons.js:2323:52
__webpack_require__@http://0.0.0.0:8000/commons.js:556:12
fn@http://0.0.0.0:8000/commons.js:87:20
./.cache/root.js@http://0.0.0.0:8000/commons.js:2026:22
__webpack_require__@http://0.0.0.0:8000/commons.js:556:12
fn@http://0.0.0.0:8000/commons.js:87:20
./.cache/app.js/<@http://0.0.0.0:8000/commons.js:912:15
run@http://0.0.0.0:8000/commons.js:6019:23
notify/<@http://0.0.0.0:8000/commons.js:6032:31
flush@http://0.0.0.0:8000/commons.js:5506:10
```

Removing the import statement fixes the issue.

**Comment by Sorgrum** [comment]
I've looked further into this and it seems that a new version of React removed PropTypes and moved them into their own package. I've fixed the issues on a branch and I'm going to create a pull request with the fixes.

**Comment by zackargyle** [comment]
Thanks for reporting! Should be fixed in 1.6.0 🤞🏼
